.pre-commit-config.yaml

```diff
@@ -95,11 +95,14 @@ repos:
           - mdformat-gfm
           #- mdformat-black
           - mdformat_frontmatter
-        exclude: src/pytorch_lightning/CHANGELOG.md
+        exclude: |
+            (?x)^(
+                src/pytorch_lightning/CHANGELOG.md|
+                src/lightning_app/CHANGELOG.md
+            )$
 
   - repo: https://github.com/PyCQA/flake8
     rev: 4.0.1
     hooks:
       - id: flake8
         name: Check PEP8
-        exclude: docs/source-app
```

Comment:
@tchaton This was added on purpose to skip pre-commit on docs ?

Reply:
well, he wanted to not break his short line code examples, but still, we shall follow PEP8 and this just raises warning does not change anything automatically